python/nano/src/bigdl/nano/tf/keras/inference/optimizer.py

```diff
@@ -514,6 +514,7 @@ def quantize(model: Model,
                                         config=openvino_config,
                                         logging=logging,
                                         **kwargs)
+            patch_compiled(result, model)
             return patch_attrs(result, model)
```

Comment:
Shall we combine these methods into one? Otherwise it's hard to enforce this in the long term.

Reply:
yeah, you are right. Will construct a new function containing these two functions.